client/src/App.js

```diff
@@ -1,15 +1,20 @@
-import { Route, Routes } from "react-router-dom";
+import { Routes, Route } from "react-router-dom";
 import Home from "./pages/Home";
 import ContactUs from "./pages/ContactUs";
 import Header from "./Components/Header";
+import Footer from "./Components/Footer";
+import Dashboard from "./pages/Dashboard";
+
 
 const App = () => (
 	<div>
-	<Header/>
-	<Routes>
-		<Route path="/" element={<Home />} />
-		<Route path="/contact-us" element={<ContactUs />} />
-	</Routes>
+		<Header />
+		<Routes>
+			<Route path="/" element={<Home />} />
+			<Route path="/dashboard" element={<Dashboard />} />
+			<Route path="/contact-us" element={<ContactUs />} />
+		</Routes>
+		<Footer />
```

Comment:
do we want this much indentation?

Reply:
Manually adjusted  indentation until I can fix eslint.

Reply:
Need to fix eslint.